HSTracker/Hearthstone/Secrets/SecretsManager.swift

```diff
@@ -162,7 +162,7 @@ class SecretsManager {
 
         if defender.isHero {
             if !fastOnly {
-                if freeSpaceOnBoard {
+                if freeSpaceOnBoard && attacker.health >= 1 {
```

Comment:
Is this for the case where the minion dies before the secret has a chance to trigger ? (typical case is explosive trap triggers before bear/freezing)
If yes, do you mind adding a comment so we can remember this case ?

Reply:
Yes, it is. I'll add a comment here.